src/server/status/metrics.js

```diff
@@ -23,5 +25,21 @@ module.exports = function (kbnServer, server, config) {
       requestsPerSecond: requestsPerSecond
     });
 
+    // Creates a status object for this node
+    kbnServer.stats = {
+      load: event.osload,
```

Comment:
how would you feel about making these field names consistent with the monitoring format?  i could see us eventually consolidating these.